<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Client</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(34, 34, 46);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="IPEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>180</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #22222e;
border: 2px solid #f66867;
border-radius: 25;
color: white</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="ConnectToServerButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>560</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #f66867;
color: white</string>
    </property>
    <property name="text">
     <string>Connect to Server</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="PortEdit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>30</y>
      <width>180</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #22222e;
border: 2px solid #f66867;
border-radius: 25;
color: white</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
    <property name="cursorMoveStyle">
     <enum>Qt::VisualMoveStyle</enum>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="ReconnectionEdit">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>30</y>
      <width>180</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #22222e;
border: 2px solid #f66867;
border-radius: 25;
color: white</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="RequestTimeoutEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>270</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #22222e;
border: 2px solid #f66867;
border-radius: 25;
color: white</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="TimeSleepEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>260</y>
      <width>270</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #22222e;
border: 2px solid #f66867;
border-radius: 25;
color: gray</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="SendRequestButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>560</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #f66867;
color: white</string>
    </property>
    <property name="text">
     <string>Send request</string>
    </property>
   </widget>
   <widget class="QLabel" name="RequestLogLabel">
    <property name="geometry">
     <rect>
      <x>238</x>
      <y>405</y>
      <width>124</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white</string>
    </property>
    <property name="text">
     <string>Request log</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>170</y>
      <width>331</width>
      <height>76</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>8</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QRadioButton" name="FastRequestRadioButton">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <strikeout>false</strikeout>
         <stylestrategy>PreferAntialias</stylestrategy>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #22222e;
color: white;</string>
       </property>
       <property name="text">
        <string>Request for fast response</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="SlowRequestRadioButton">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <strikeout>false</strikeout>
         <stylestrategy>PreferDefault</stylestrategy>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #22222e;
color: white</string>
       </property>
       <property name="text">
        <string>Request for slow response</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>440</y>
      <width>400</width>
      <height>300</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #22222e;
border: 3px solid #f66867;
color: white</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
